docs/book/v6/introduction/packages.md

```diff
@@ -0,0 +1,27 @@
+# Packages
+
```

Comment:
Add the same (adapted to Admin) message we used in the [API docs](https://docs.dotkernel.org/api-documentation/v6/introduction/packages/).